src/renderers/shared/fiber/ReactFiberContext.js

```diff
@@ -57,6 +57,15 @@ function getUnmaskedContext(workInProgress : Fiber) : Object {
 }
 exports.getUnmaskedContext = getUnmaskedContext;
 
+function cacheContext(workInProgress : Fiber, unmaskedContext : Object, maskedContext : Object) {
+  const instance = workInProgress.stateNode;
+  if (instance) {
```

Comment:
Can we move this check in `getMaskedContext` instead?
The other caller of `cacheContext` always has the instance.